orchestra/bots/staffbot.py

```diff
@@ -34,7 +34,7 @@
 # A regex for markdown links, which are of the form [link text](url).
 # We capture link text and url as an text that isn't a closing ] or )
 # respectively.
-MARKDOWN_LINK_REGEX = re.compile('\[(?P<text>[^\]]*)\]\((?P<url>[^\)]*)\)')
+MARKDOWN_LINK_REGEX = re.compile('\[(?P<text>[^\]]*)\]\((?P<url>[^\)]*)\)')  # noqa
```

Comment:
For the regex noqas, I remember when upgrading the other codebase to Python 3.8 I got a bunch of escape sequence warnings. I believe prefixing them with `r` did the trick, but this has a bit more details: https://stackoverflow.com/questions/52335970/how-to-fix-string-deprecationwarning-invalid-escape-sequence-in-python

Reply:
@marcua It worked!